docs/developer/decision-records/2022-06-15-registration-service/README.md

```diff
@@ -0,0 +1,147 @@
+2. The Participant A Identity Hub contains VCs that satisfy the Participant B's service access policy.
+
+#### Flow sequence
+
+![distributed-authorization](distributed-authorization.png)
+
+1. The Client for Participant A (which could be EDC, or any other application) sends a request to Participant B's API. The client needs access to Participant A's Private Key to sign a JWS. It also sends a time-limited bearer token granting access to its Identity Hub.
+2. Participant B retrieves the DID Document based on the DID URI contained in the JWS.
+3. Participant B authenticates the request by validating the JWS signature against the public key in the DID Document.
+4. Participant B finds Participant A's Identity Hub URL in the DID Document. It authorizes the request by obtaining VCs for Participant A at its Identity Hub, 
+   using the bearer token sent initially by Participant A.
+5. Participant A's Identity Hub verifies the bearer token validity.
+6. Participant A's Identity Hub returns Participant A's Verifiable Credentials.
+7. Participant B applies its access policy for the given service. This applies expiration dates and Certificate Revocation Lists to filter valid Verifiable 
+   Credentials, and rules specific to a given service. For example, the caller must be a dataspace participant (i.e. have a valid Verifiable Credential signed by the Dataspace Authority, that establishes its dataspace membership).
+8. Participant B returns the service response if the request was successfully authorized, otherwise, an error response. Depending on the flow, the response can be synchronously or asynchronously returned.
+
+### 2. Dataspace participant enrollment
+
+#### Participants
+
+1. Company1, an entity which intends to become a Dataspace participant
+2. The Dataspace Authority, which manages the enrollment process
+
+#### Overview
+
+A Client for Company1 initiates the enrollment process by resolving and contacting the enrollment API endpoint for the Dataspace Authority. The client could be e.g. a CLI utility.
+
+The Dataspace Authority enrollment service obtains Verifiable Credentials from Company1 to determine whether it meets enrollment policies. The enrollment service then issues a Verifiable Credential that establishes membership and pushes it to Company 1's Identity Hub, and stores membership and certificate information.
+
+In simple scenarios, enrollment could be fast and fully automated. However, in advanced scenarios, enrollment policies could require interactions with external systems, and even manual processes. Therefore, it is implemented asynchronously.
+
+#### Pre-conditions
+
+1. Company1 has deployed an Identity Hub service, and a DID Document containing the Identity Hub URL.
+2. Company1 knows the DID URL of the Dataspace it intends to join.
+3. The Company1 Identity Hub contains VCs that satisfy the Dataspace Authority enrollment policy. For example, it could be a credential signed by the German 
```

Comment:
```suggestion
3. The _Company1_ Identity Hub contains VCs that satisfy _the Dataspace Authority_ enrollment policy. For example, it could be a credential signed by the German 
```